src/test/java/uk/gov/laa/pfla/ServiceManager.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.laa.pfla;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ServiceManager {
+    private static Process serviceProcess;
+
+    public static void startService() throws IOException {
+        String jarPath = new File("target/libs/pay-for-legal-aid-0.0.1-SNAPSHOT.jar").getAbsolutePath();
+        ProcessBuilder processBuilder = new ProcessBuilder("java", "-jar", "-Dspring.profiles.active=local", jarPath);
+        processBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
+        processBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
+        serviceProcess = processBuilder.start();
+    }
+
+    public static void stopService() {
+        if (serviceProcess != null && serviceProcess.isAlive()) {
+            serviceProcess.destroy();
```

Comment:
**Suggestion**: Add a more robust shutdown mechanism for the service by using destroyForcibly() https://docs.oracle.com/javase/8/docs/api/java/lang/Process.html#destroyForcibly and a shutdown hook.
**Reason**:
Using destroyForcibly() ensures that the service is properly terminated, even if it becomes unresponsive. Additionally, a shutdown hook ensures that resources are released even if the JVM exits unexpectedly: